packages/animate/README.md

```diff
@@ -1,3 +1,31 @@
+# @gov.au/animate
+
+> A JavaScript function to open, close and toggle the display of page elements.
+
+## Contents
+
+- [Install](#install)
+- [API](#api)
+- [Dependency graph](#dependency-graph)
+- [Tests](#tests)
+- [Release History](#release-history)
+- [License](#license)
+
+---
+
+## Install
+
+```shell
+yarn add @gov.au/animate
+```
+
+```shell
+npm install @gov.au/animate
```

Comment:
The only difference in this file should be the release history and install instruction? We seem to have some spacing changes that make this PR harder to read